resources/views/layout/metadata.blade.php

```diff
@@ -28,6 +28,20 @@
 <meta name="keywords" content="osu, peppy, ouendan, elite, beat, agents, ds, windows, game, taiko, tatsujin, simulator, sim, xna, ddr, beatmania, osu!, osume">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
+@if ($hasOpengraph)
+    <meta property="og:site_name" content="{{ 'osu! » '.$opghSitename ?? 'osu!' }}">
```

Comment:
I think it should always be 'osu!'.

Reply:
wouldnt it be less misleading if for title "Ben Briggs", it would show `osu! » featured artists`?